loop_rate:        1.5    # The frequency in Hz at which to run the control loops.
queue_size:       10     # The size of the message queue used for publishing and subscribing to topics.
swarm_timeout:    5.0    # The time in seconds to wait after an area division event before starting the area division.
visualize:        false  # Whether to publish the area division on a topic for visualization.
optimizer:
  iterations:     100    # Maximum number of iterations of the optimization algorithm.
  variate_weight: 0.01   # Maximum variate weight of connected components.
  discrepancy:    30     # Maximum difference between number of assigned grid cells to each CPS.
